js/components/base-components/listItem.js

```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+var Icon = require('../icon');
+var classnames = require('classnames');
+
+var listItem = React.createClass({
+  handleSelect: function () {
```

Comment:
Is this handleAudioSelect? If it is, please rename

Reply:
it is base component